lib/OnyxConnectionManager.ts

```diff
@@ -89,7 +95,10 @@ class OnyxConnectionManager {
      */
     private generateConnectionID<TKey extends OnyxKey>(connectOptions: ConnectOptions<TKey>): string {
         const {key, initWithStoredValues, reuseConnection, waitForCollectionCallback} = connectOptions;
-        let suffix = '';
+
+        // The current session ID is appended to the connection ID so we can have different connections
+        // after an Onyx.clear() operation.
```

Comment:
>so we can have different connections after an Onyx.clear() operation.

Let's also say why that is important or needed?

Reply:
I moved all explanations to `private sessionID: string;`, please have a look!